.github/workflows/bindings.yml

```diff
@@ -17,6 +17,7 @@ concurrency:
 jobs:
   go:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
```

Comment:
remove?

Reply:
Tbh I kinda misread this line at first. Ideally we'd be doing the same thing as the monorepo with draft + `[ci-run]` so  let's remove from this PR and I'll open an issue to do it properly.